pkg/operation/garden/garden.go

```diff
@@ -191,7 +201,13 @@ func ReadGardenSecrets(ctx context.Context, c client.Reader, namespace string, l
 		if secret.Labels[v1beta1constants.GardenRole] == v1beta1constants.GardenRoleDefaultDomain {
 			_, domain, _, _, _, err := gutil.GetDomainInfoFromAnnotations(secret.Annotations)
 			if err != nil {
-				log.Warnf("error getting information out of default domain secret %s: %+v", secret.Name, err)
+				switch logger := log.(type) {
+				case logrus.FieldLogger:
+					logger.Warnf("error getting information out of default domain secret %s: %+v", secret.Name, err)
+				case logr.Logger:
+					logger.Info("Error getting information out of default domain secret", "secret", client.ObjectKeyFromObject(&secret))
```

Comment:
Shouldn't we use `logger.Error()` here for consistency to also add the error message to the log?